pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -268,6 +270,8 @@ func convertHTTPRoute(r k8s.HTTPRouteRule, ctx configContext,
 			}
 		case k8s.HTTPRouteFilterURLRewrite:
 			vs.Rewrite = createRewriteFilter(filter.URLRewrite)
+		case k8s.HTTPRouteFilterCORS:
```

Comment:
I don't see any conformance tests upstream, so I'd like to see some testing here to make sure we implement this correctly

Reply:
I can add some next week, though we do have unit tests converting to virtual services which have e2e tests, so we kind of have transitive e2e tests

Reply:
Ok that's ok for now I guess. My major concern was the wildcard stuff so I'm more ok with the lack of tests now that's reverted